ui/components/app/snaps/snap-authorship-header/snap-authorship-header.js

```diff
@@ -57,13 +50,15 @@ const SnapAuthorshipHeader = ({
   return (
     <Box
       className={classnames('snaps-authorship-header', className)}
-      backgroundColor={BackgroundColor.backgroundAlternative}
+      backgroundColor={BackgroundColor.backgroundDefault}
       width={BlockSize.Full}
       alignItems={AlignItems.center}
       display={Display.Flex}
       padding={4}
       style={{
         boxShadow,
+        minHeight: '64px',
```

Comment:
Curious what this is needed for 🤔

Reply:
@eriknson requested that header height should strictly be `64px`. It was usually less than 64px because content was not enough to push it down a bit.